packages/styles/gulpfile.js

```diff
@@ -30,6 +30,21 @@ gulp.task('temporarily-copy-token-files', () => {
   return gulp.src(['../tokens/dist/**/*.scss']).pipe(gulp.dest('./src/tokens/temp'));
 });
 
+/**
+ * Temporary task to copy icon CSS files from icons package to the styles package since
+ * pnpm does not correctly install dependencies of dependencies for workspace packages.
+ */
+gulp.task('temporarily-copy-icon-files', () => {
+  return gulp.src(['../icons/dist/custom-properties/**/*.css']).pipe(gulp.dest('./src/icons/temp'));
+});
+
+/**
+ * Copy icon CSS files to dist folder
+ */
+gulp.task('copy-icon-files-to-dist', () => {
+  return gulp.src(['./src/icons/temp/**/*.css']).pipe(gulp.dest(`${options.outputDir}/icons/temp`));
+});
```

Comment:
This should be executed, whenever a the copy task is executed.

Somehow like this:
```js
gulp.task('copy', () => {
  gulp.series('copy-icon-files-to-dist', () => {
    return gulp
      .src(['./LICENSE', './README.md', './package.json', './src/**/*.scss'])
      .pipe(newer(options.outputDir))
      .pipe(gulp.dest(options.outputDir));
  });
});
```